docs/src/domains/cdn/_index.md

```diff
@@ -17,41 +17,43 @@ By default, Dedicated projects include a [Fastly CDN managed by Platform.sh](./m
 Self-service Grid plans don't include a CDN by default, but you can set up one at any time,
 such as [Fastly](./fastly.md) or [Cloudflare](./cloudflare.md).
 
-## `CNAME` records
+## DNS records
 
 To start routing client traffic through your CDN,
-create `CNAME` records for your domain names through your DNS provider.
-For more information, see you DNS provider's official documentation.
+several specific records on your DNS are required.
+These usually are the `_acme-challenge` subdomain and `CNAME` records.
+You can create these records for your domain names through your DNS provider.
+For more information, see you DNS and your CDN provider's official documentations.
```

Comment:
🐞 The phrasing here is confusing to me because my understanding is that a subdomain isn't a type of DNS record. Did you mean that users need to create a `CNAME` record for the `_acme-challenge` subdomain? I may be completely missing the point here but I think it's worth clarifying.